server/mqtt.go

```diff
@@ -893,6 +917,14 @@ func (s *Server) mqttCreateAccountSessionManager(acc *Account, quitCh chan struc
 			nuid:   nuid.New(),
 			quitCh: quitCh,
 		},
+		sp: sessPersist{
+			ch: make(chan struct{}, 1),
+		},
+	}
+	// We need to include the domain in the subject prefix used to store sessions in the $MQTT_sess stream.
+	// If no domain is present, use "_" so that the token always exists.
```

Comment:
Where do we do the "_" part?

Reply:
That is left over comment before I decided to not go with "_.".
Also, I realize that below I am using `getHash()` for the domain, and I am not sure I should do that. It may be ok (or even a good thing) to see what the domain is in this subject.

Reply:
ok sounds reasonable.